### PR TITLE
[dist] Adapt installation paragraph in README.md

### PR DESCRIPTION
**Pull request by hennevogel**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,39 +24,30 @@ The OBS source code repository is hosted on [Github](http://github.com/opensuse/
...
+    ```shell
+    zypper ar -f http://download.opensuse.org/repositories/OBS:/Server:/2.7/openSUSE_42.1/OBS:Server:2.7.repo
+    ```
 
-  After finishing the installation of your base system, follow the following steps:
+2. Install the package
 
-  1. Add the OBS software repository with zypper
+   ```shell
+   zypper in obs-api obs-server
```
Comment (adrianschroeter) [inline README.md]: We should either add the optional packages here or point to the pattern.

Additional packages are:   obs-source_service obs-signd

Alternative solution would be to use the pattern:

 zypper in -t pattern OBS_Server

Maybe the better solution, so the documentation can stay unchanged on package changes.

File: README.md [hunk 2]
```diff
@@ -24,39 +24,30 @@ The OBS source code repository is hosted on [Github](http://github.com/opensuse/
...
-
-    If you have a more complex setup (e.g. a distributed backend) we recommend to follow our [Manual Setup Guide](dist/README.SETUP.md)
+3. Run our setup wizard
 
+   ```shell
+   /usr/lib/obs/server/setup-appliance.sh
+   ```
 
+If you have a more complex setup (e.g. a distributed backend) we recommend to read the Administration
+chapter in our [reference manual](http://openbuildservice.org/help/manuals/obs-reference-guide/cha.obs.admin.html).
```
Comment (adrianschroeter) [inline README.md]: Hm, not much there yet and there is (hopefully) also an own admin book comming, so this url would not be valid anymore. I would drop that part here for now, we can re-add it later again.

Comment (hennevogel) [inline README.md]: See https://github.com/openSUSE/obs-docu/pull/26

**Comment by adrianschroeter** [comment]
manual merged to solve conflicts.

**Pull request closed without merging**